data/getdata.sh

```diff
@@ -7,8 +7,8 @@ echo "Unzipping the data files (overwrites any existing .tsv files)..."
 gunzip -vf *.gz
 echo
 echo "Removing IMDb '\N' values..."
-sed --in-place 's/\\N//g' *.tsv
+ed -s *.tsv <<< 's/\\N//g'
```

Comment:
did you mean `sed` here?

Reply:
ed is another software for linux that can do replacement.  Its very fast to do these conversions compared with sed.  Its reduced minutes of waiting to seconds.